### ISSUE TITLE
(New feature) add a tickets section

### ISSUE THREAD

**Issue by Ali-Asim800** [Huntereb/Awoo-Installer#74]
Please add tickets section to view all of the tickets, I would really like like awoo-installer more if there was a tickets section, The tickets should be displayed with the title id's and name of the game that the ticket belongs to. It would be very convient adding the name of the game that corresponds to the ticket and would make things much much easier. I hope this gets added.

**Comment by Huntereb** [comment]
It's a possibility, but I only really intended Awoo to be an installer. Does one thing well, ya know?

**Comment by Ali-Asim800** [comment]
yeah i get it and also a possiblity as you said, i'm just saying that it would be really nice adding this feature. Think of it this way, awoo is a installer, you install tickets along with the nsp/xci you install, you want to view those tickets and updates and dlc. you get it, so that way it kinda fits in. Whats the point of having the dlc when you can't view the tickets that you got from them? just saying it would make awoo installer better, alot better in my terms of view.
